dev/tests/static/framework/Magento/ruleset.xml

```diff
@@ -85,4 +85,8 @@
     <rule ref="Squiz.Commenting.DocCommentAlignment"/>
     <rule ref="Squiz.Functions.GlobalFunction"/>
     <rule ref="Squiz.WhiteSpace.LogicalOperatorSpacing"/>
+    <rule ref="Squiz.Operators.ValidLogicalOperators">
+        <severity>6</severity>
```

Comment:
No need to spoil core coding standard with severities, each rule is mandatory.

Please put it before previous line to preserve alphabetical order.

Reply:
Done